scene/2d/navigation_agent_2d.h

```diff
@@ -141,10 +141,10 @@ class NavigationAgent2D : public Node {
 
 	Vector2 get_next_location();
 
-	const Vector<Vector2> &get_nav_path() const;
+	const Vector<Vector2> &get_current_navigation_path() const;
 
-	int get_nav_path_index() const {
-		return nav_path_index;
+	int get_navigation_path_index() const {
```

Comment:
```suggestion
	int get_current_navigation_path_index() const {
```

For consistency with `get_current_navigation_path`.